src/Microsoft.Framework.Configuration.FileExtensions/FileConfigurationExtensions.cs

```diff
@@ -49,7 +49,12 @@ public static string GetBasePath(this IConfigurationBuilder configurationBuilder
                 return (string)basePath;
             }
 
-            return string.Empty;
+#if DOTNET
+            return AppContext.BaseDirectory;
```

Comment:
I don't think we have a reason to trust this one that never return null (the implementation usually calls `AppDomain.CurrentDomain.BaseDirectory`) so I would add `?? string.Empty` here as well.